File: app/src/main/java/ru/rodionov/aleksey/picchan/PicChanApp.java
package ru.rodionov.aleksey.picchan;

import android.app.Application;

import timber.log.Timber;

public class PicChanApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: app/src/main/java/ru/rodionov/aleksey/picchan/PicChanApp.kt
package ru.rodionov.aleksey.picchan

import android.app.Application
import android.content.Context
import timber.log.Timber

class PicChanApp : Application() {

    override fun onCreate() {
        super.onCreate()

        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
    }

}
